src/react/components/common/assetPreview/imageAsset.tsx

```diff
@@ -12,7 +12,8 @@ export class ImageAsset extends React.Component<IAssetProps> {
             <img ref={this.image}
                 src={this.props.asset.path}
                 onLoad={this.onLoad}
-                onError={this.props.onError} />);
+                onError={this.props.onError}
+                crossOrigin="anonymous" />);
```

Comment:
What is this for?

Reply:
To allow to call canvas.getBlob in the browser